### PR TITLE
build: remove dev-infra directory and migrate to relying on @angular/dev-infra-private-builds

### PR DESCRIPTION
**Pull request by josephperrott**
Removing dev-infra/ as its contents have been migrated to angular/dev-infra

### CODE REVIEW

File: tools/postinstall-patches.js [hunk 1]
```diff
@@ -61,6 +61,20 @@ ls('node_modules/@types').filter(f => f.startsWith('babel__')).forEach(pkg => {
   }
 });
 
+log('\n# patch: use local version of @angular/* and zone.js in component_benchmark from @angular/dev-infra.-private');
+[['@npm//@angular/platform-browser', '@angular//packages/platform-browser'],
+ ['@npm//@angular/core', '@angular//packages/core'],
+ [
+   'load\\("@npm//@angular/bazel:index.bzl". "ng_module"\\)',
```
Comment (devversion) [inline tools/postinstall-patches.js:68]: why are we using `.` instead of the comma here? would the comma work too?

**Review by devversion — APPROVED** [review]

File: tools/postinstall-patches.js (hunk 1 above)
Comment (josephperrott) [reply tools/postinstall-patches.js]: Yep, errantly used a `.` instead of `,`

Comment (devversion) [reply tools/postinstall-patches.js]: 👍

**Comment by dylhunn** [comment]
This doesn't seem to merge cleanly into the patch branch, mind having a look?

**Comment by josephperrott** [comment]
@dylhunn We can merge this as `major` and I will create a `patch` PR as well

**Pull request closed without merging**